### ISSUE TITLE
Setting to have Friends list open automatically

### ISSUE THREAD

**Issue by Zenzi0** [ValveSoftware/steam-for-linux#9785]
Not really a bug but a feature request.
I'm running a window manager (Sway) and use the keyboard to close windows and the Steam client as well. It used to be that when I closed first the main window and then the friends list and then launched Steam the next time the friends list would automatically re-open. Now with Steam closing to the system tray this behaviour has changed I have to open the friends list manually unless I quit Steam with the mouse via `Steam >> Exit`. 
So basically I would like the old behaviour restored where I could simply close the friends list last (without having to use the mouse) and it would open automatically when launching it the next time. Maybe a setting to toggle whether Steam should close to the system tray would do it?
